epub33/core/index.html

```diff
@@ -4873,7 +4873,7 @@ <h4>The <code>page-list nav</code> Element </h4>
 
 					<p>The <code>page-list</code>
 						<code>nav</code> element provides navigation to positions in the content that correspond to the
-						locations of page boundaries present in a print source.</p>
+						locations of page boundaries present in a print source or other static source.</p>
```

Comment:
This paragraph is kind of a mouthful. It also sounds like it excludes page lists without a source.

Maybe something like the following would be better here:

> The `page-list` `nav` element provides navigation to virtual page boundaries in the content. These boundaries may correspond to a statically paginated source such as print or may be defined exclusively for the EPUB Publication." 

(Any thoughts on this @avneeshsingh @GeorgeKerscher @clapierre @gregoriopellegrino ?)

Reply:
Does silence mean everyone is good with this rewrite of the page-list nav description?

Reply:
I am fine with it

Reply:
(I have updated the PR accordingly.)